Documentation/UseCases/Index.rst

```diff
@@ -272,3 +272,49 @@ like `https://example.com/downloads/detail/some-download/hz6dFgz9/`, where
 
 .. _custom dimension: https://matomo.org/docs/custom-dimensions/
 .. _setting a custom URL: https://matomo.org/faq/how-to/how-do-i-set-a-custom-url-using-the-matomo-javascript-tracker/
+
+Extending the script tag
+========================
+
+Using tracking tools like Matomo within the European Union need special treatments in order
+to let the customer consent and agree with the tracking. Although Matomo respects the browser's "Do not track" setting, not everyone is aware of it.
+
+Some GDPR tools like klaro.js require special attribute settings within the script tag in order to work.
+
+The event listener
+------------------
+
+   Before the script tag is rendered the event `EnrichScriptTagEvent` dispatched from the injector.
+   This events allow to register an id, a type and add additional data attributes.
+
+   ::
+
+      <?php
+      declare(strict_types=1);
+
+      namespace YourVendor\YourExtension\EventListener;
+
+      use Brotkrueml\MatomoIntegration\Event\EnrichScriptTagEvent;
+
+      final class PrepareScriptTagForKlaroJs
+      {
+         public function __invoke(EnrichScriptTagEvent $event)
+         {
+            $event->setType('text/plain');
+            $event->addDataAttribute('type', 'application/javascript');
+            $event->addDataAttribute('name', 'matomo');
+         }
+      }
+
+
+#. Registration of the event listener in :file:`Configuration/Services.yaml`
+
+   .. code-block:: yaml
+
+      YourVendor\YourExtension\EventListener\PrepareScriptTagForKlaroJs:
+         tags:
+            - name: event.listener
+              identifier: 'your-ext/prepare-script-for-klaro-js'
+              event: Brotkrueml\MatomoIntegration\Event\EnrichScriptTagEvent
```

Comment:
Please add also the comment `# The event tag can be omitted for TYPO3 v11+` before `event:`. Added it some days ago.

Reply:
Added